src/Nethermind/Nethermind.Xdc/Spec/XdcReleaseSpec.cs

```diff
@@ -32,6 +32,8 @@ public class XdcReleaseSpec : ReleaseSpec, IXdcReleaseSpec
 
     public Address[] GenesisMasterNodes { get; set; }
 
+    public Address MasternodeVotingContract { get; set; }
```

Comment:
Should this be set through the CreateReleaseSpec in XdcChainSpecBasedSpecProvider?
Also is this a field we need to add to the xdc.json configuration?